data_structures/quadTree/draw.py

```diff
@@ -0,0 +1,45 @@
+import turtle
```

Comment:
Please change the directory from __quadTree__ to __quad_tree__ as discussed in CONTRIBUTING.md.